src/app/pages/Designathons/Designathon24/Designathon24.js

```diff
@@ -1,13 +1,12 @@
 import { Icon, Section } from "app/Symbols";
 import { Text } from "../../../components";
```

Comment:
Chore: to avoid traversing the tree upwards, could we use an absolute import instead of relative?
```suggestion
import { Text } from "app/components";
```